taskcluster/fenix_taskgraph/worker_types.py

```diff
@@ -157,3 +157,56 @@ def build_github_release_payload(config, task, task_def):
         "{}:github:project:{}".format(scope_prefix, worker["github-project"]),
         "{}:github:action:{}".format(scope_prefix, worker["action"]),
     ])
+
+
+@payload_builder(
+    "scriptworker-tree",
+    schema={
+        # FIXME: do we need upstream artifacts?
+        Required("upstream-artifacts"): [
+            {
+                Required("taskId"): taskref_or_string,
+                Required("taskType"): text_type,
+                Required("paths"): [text_type],
+            }
+        ],
+        Optional("dontbuild"): bool,
```

Comment:
I'd say: no need to support `DONTBUILD` yet. It's just supported for HG, so far https://hg.mozilla.org/ci/taskgraph/file/4c4d72d32c5ec36a3d0187bf792ac08f1348f9fa/src/taskgraph/decision.py#l190